projects/packages/assets/src/class-assets.php

```diff
@@ -112,7 +114,17 @@ public function script_add_async( $tag, $handle ) {
 	 */
 	public static function enqueue_async_script( $handle, $min_path, $non_min_path, $deps = array(), $ver = false, $in_footer = true ) {
 		$assets_instance = self::instance();
-		$assets_instance->add_async_script( $handle );
+
+		// WordPress 6.3 introduces support for async/defer via the strategy attribute.
+		if ( version_compare( get_blog_info( 'version' ), '6.3', '>=' ) ) {
```

Comment:
> Why get_blog_info( 'version' ) rather than comparing with $wp_version like we do almost everywhere else?

I missed that was your approach in the codebase, either approach is valid. I will update here to match what you use.

Reply:
updated in caaec1424a